appveyor-windows-binary.yml

```diff
@@ -49,10 +49,6 @@ init:
   - ps: New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
   - ps: git config --system core.longpaths true
 
-cache:
-  - C:\ProgramData\chocolatey\bin -> appveyor.yml
-  - C:\ProgramData\chocolatey\lib -> appveyor.yml
-
```

Comment:
out of curiosity, why did we removed the caching part here?

Reply:
This doesn't work for a couple of reasons:
1. The cache works by specifying a cached directory, and file to watch to invalidate the cache. We don't have a file named `appveyor.yml`, so the cache will never be invalidated correctly.
2. Its not enough to just watch the chocolatey directories as some tools we install via chocolatey dump their files in other locations. For example, we install JDK 21 via chocolatey, but the runtime is saved into `C:\Program Files\...`, while the binaries and chocolatey metadata is saved in the chocolatey folders. This causes a "desync" and will cause the job to fail.

We're going to disable it for now since this needs some more investigation/work before turning it back on.